helm/repo/index.yaml

```diff
@@ -20,34 +20,13 @@ entries:
     sources:
     - https://github.com/kubernetes-sigs/cloud-provider-azure
     urls:
-    - cloud-provider-azure-1.24.0+20220601.tgz
-    version: 1.24.0+20220601
+    - cloud-provider-azure-1.24.1.tgz
+    version: 1.24.1
   - apiVersion: v1
-    created: "2022-05-26T13:57:34.35813407+02:00"
-    description: A Helm chart for installing kubernetes-sigs/cloud-provider-azure
-      components
-    digest: 90de09f45ccf3a379eb2959236244670ee0fbf66ecf9ec056c9b39050bf69646
-    home: https://raw.githubusercontent.com/kubernetes-sigs/cloud-provider-azure/master/helm/cloud-provider-azure/README.md
-    keywords:
-    - cloud-provider-azure
-    - kubernetes-sigs
-    - kubernetes
-    maintainers:
-    - email: jackfrancis@gmail.com
-      name: Jack Francis
-    - email: zhechengli1995@outlook.com
-      name: Zhecheng Li
-    name: cloud-provider-azure
-    sources:
-    - https://github.com/kubernetes-sigs/cloud-provider-azure
-    urls:
-    - cloud-provider-azure-1.24.0+20220526.tgz
-    version: 1.24.0+20220526
```

Comment:
Maybe we can manually keep 1.24.0, 1.24.0+20220526?

Reply:
Agreed, done.